developer/app-router/installation.mdx

```diff
@@ -85,10 +85,10 @@ Create the Makeswift [client](/developer/reference/client/constructor) file in `
   
 Similar to [NextAuth.js](https://next-auth.js.org/), Makeswift uses an API handler to communicate with your Next.js app. Create the file `src/app/api/makeswift/[...makeswift]/route.ts`.
 
-<Info>
+<Note>
```

Comment:
`<Note>` has a more prominent appearance, which is more appropriate here. Matching a corresponding change in the Pages Router guide.

**Before**:

<img width="770" alt="before" src="https://github.com/user-attachments/assets/567233b6-c1a7-4a67-aebb-07a78e7b3342" />

**After**:

<img width="776" alt="after" src="https://github.com/user-attachments/assets/d80f7bec-8330-4a6f-be01-81c1a3832d23" />